tests/console/validate_zfcp_multipath.pm

```diff
@@ -36,13 +36,20 @@ sub verify_plain_scsi_block_devices {
 sub verify_multipath_block_devices {
     my $mpath = script_output "multipath -l";
     my @mpath = split /\n/, $mpath;
+    my $origin_wwid = get_dynamic_wwid();
     # check for 50GB disk (multiline match)
-    assert_matches qr/^3600507638081855cd800000000000076.+\n^size=50G/m, $mpath, "Cannot find 50GB disk on WWID *076";
+    assert_matches qr/^$origin_wwid.+\n^size=50G/m, $mpath, "Cannot find 50GB disk on WWID *076";
     # must see the LUN over 2 paths
     my $disks = scalar grep { /\bsd(a|b)\b/ } @mpath;
     assert_equals 2, $disks, "50GB block device does not have 2 paths";
 }
 
+# Get the current dynamic WWID and do not rely on hard-code one.
+sub get_dynamic_wwid {
+    my $mpath = script_output "/usr/lib/udev/scsi_id --whitelisted --replace-whitespace --device=/dev/sda";
```

Comment:
The name of the variable doesn't represent what the function returns (for a 'get', should be 'wwid'), which goes in the direction that as you have one line, you don't need to create a function, but execute this inside `verify_multipath_block_devices` function.

Reply:
Moved the line to verify_multipath_block_devices.